app/src/main/java/ch/epfl/culturequest/social/Profile.java

```diff
@@ -0,0 +1,68 @@
+package ch.epfl.culturequest.social;
+
+import android.net.Uri;
+
+import com.google.firebase.auth.FirebaseUser;
+
+import java.util.Objects;
+
+/**
+ * Creates a profile for users
+ */
+public class Profile implements User {
+
+    private final User user;
+    private final String uid, fullName,firstName, lastName, email, phoneNumber;
+    private Uri profilePicture;
```

Comment:
Maybe the user should as well be able to choose a pseudo? Or do you think name is enough?